arch/arm/soc/nxp_kinetis/k6x/nxp_mpu_regions.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2017 Linaro Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <soc.h>
+#include <arch/arm/cortex_m/mpu/nxp_mpu.h>
+
+#define FLEXBUS_BASE_ADDRESS 0x08000000
+#define SRAM_L_BASE_ADDRESS 0x1FFF0000
+#define DEVICE_S_BASE_ADDRESS 0x20030000
+
+static struct nxp_mpu_region mpu_regions[] = {
+	/* Region 0 */
+	MPU_REGION_ENTRY("FLASH_0",
+			 CONFIG_FLASH_BASE_ADDRESS,
+			 0x07FFFFFF,
+			 REGION_FLASH_ATTR),
+	/* Region 1 */
+	/*
+	 * This region (Flexbus + FlexNVM) is bigger than the FLEXBUS one in
+	 * order to save 1 region allocation in the MPU.
+	 */
+	MPU_REGION_ENTRY("FLEXBUS_0",
+			 FLEXBUS_BASE_ADDRESS,
+			 0x1FFEFFFF,
```

Comment:
Per my gerrit review, I'm not sure this address is correct. I think it should be 0x1BFF_FFFF?